README.md

```diff
@@ -7,6 +7,28 @@ bells and whistles.
 For a longer description and available features and options, see the
 [man page](mkosi.md).
 
+# Installation
+
+Installing `mkosi` is easy, as it has no runtime dependencies.
```

Comment:
I'd remove this. We don't have any python dependencies but we do have a few tool dependencies. This might annoy users since they could expect mkosi to "just work" after reading this and then it subsequently fails because it can't find a mkfs or package manager program or whatnot.